src/workerd/api/tests/js-rpc-test.js

```diff
@@ -246,8 +246,8 @@ export let namedServiceBinding = {
     assert.strictEqual(sawFinally, true);
 
     // `fetch()` is special, the params get converted into a Request.
-    let resp = await env.MyService.fetch("http://foo", {method: "POST"});
-    assert.strictEqual(await resp.text(), "method = POST, url = http://foo");
+    let resp = await env.MyService.fetch("http://foo/", {method: "POST"});
```

Comment:
Out of curiosity, why is adding the trailing `/` here necessary?

Reply:
When sending the request over actual HTTP, the trailing `/` gets added if it is missing. So in order to make the test consistent between when it runs over a real socket vs. local service binding, I had to add the `/`.

Reply:
hmm, ok. Per fetch spec the trailing slash is supposed to be implicit in this case so it technically should be fine to pass in without it (since the `url` arg here is technically supposed to be passed through the whatwg url parser if we're following spec)... so it just feels a bit odd to have to modify the test to make it work correctly. But I think this is fine for now. I do think at some point we need to revisit how the input url is handled on fetch calls tho.

Reply:
The problem is if I don't add the `/` here, then the assert on the next line fails either when using real HTTP (if it doesn't expect a `/`) or when not using real HTTP (if it does expect a `/`). In order to be able to write a single assert which works in both cases, I had to add the `/` on this line.

Arguably this is a bug in service bindings, that they do not implicitly add the `/` as regular HTTP would, but that bug has been around a while, not introduced in this PR.